### ISSUE TITLE
start sample_app.py error

### ISSUE THREAD

**Issue by Linkding** [rochacbruno/flask_simplelogin#5]
i got the error when i try to run this app
```
#python simple_app.py 
Traceback (most recent call last):
  File "simple_app.py", line 39, in <module>
    @login_required(username=['chuck', 'mary'])
  File "/usr/local/python2.7/lib/python2.7/site-packages/flask_simplelogin/__init__.py", line 109, in login_required
    @wraps(function)
  File "/usr/local/python2.7/lib/python2.7/functools.py", line 33, in update_wrapper
    setattr(wrapper, attr, getattr(wrapped, attr))
AttributeError: 'NoneType' object has no attribute '__module__'
```

my env python version is 
```
# python -V
Python 2.7.13 (default, Aug  9 2017, 23:25:57) 
```
OS system: ** centos 6.5 64bit**

then i also try run `python manage.py runserver`

```
Traceback (most recent call last):
  File "manage.py", line 135, in <module>
    main()
  File "/usr/local/python2.7/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/python2.7/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/python2.7/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/python2.7/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/python2.7/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "manage.py", line 89, in decorator
    configure_views(app)
  File "manage.py", line 66, in configure_views
    @login_required()
  File "/usr/local/python2.7/lib/python2.7/site-packages/flask_simplelogin/__init__.py", line 109, in login_required
    @wraps(function)
  File "/usr/local/python2.7/lib/python2.7/functools.py", line 33, in update_wrapper
    setattr(wrapper, attr, getattr(wrapped, attr))
AttributeError: 'NoneType' object has no attribute '__module__'
```

seems have the same problem

**Comment by Linkding** [comment]
when i remove the code behide `@login_required`, then it seems work well

**Comment by claytonaalves** [comment]
Same problem here.

Python 2.7.13 on Windows.

**Comment by rochacbruno** [comment]
Looks like `wraps` works different in Py 2.7, I'll try a fix for this as I only tested in Python 3.6+

**Comment by rochacbruno** [comment]
Fixed in 39435c0737f51e26bda033923983c267a67f9f6a
